README.md

```diff
@@ -1,3 +1,82 @@
 # Noflake
 
-A web service to automatically detect and track your flaky tests based on junit reports.
+A web service that automatically detects and tracks flaky tests based on junit
+reports.
+
+## How it works
+
+After every CI build of your software, you upload all junit reports together
+with the commit SHA of the test run. Noflake keeps track of the results and
+automatically detects flakiness when a test passes and fails on the same
+commit. The Noflake UI shows you all detected flaky tests in a given time range
+sorted by the number of fails relative to the total test invocations.
+
+## How to use
+
+1. Run Noflake:
+   ```bash
+    docker run \
+        -v $PWD/noflake.sqlite3:/opt/noflake/noflake.sqlite3 \
```

Comment:
Doesn't work if you noflake.sqlite3 doesn't exist yet. Docker then creates an empty directory.

Reply:
Fixed! I'll release 0.1.0 and update the ansible deployment script after merging.